package.json

```diff
@@ -51,6 +51,7 @@
   },
   "dependencies": {
     "classnames": "2.2.6",
+    "cozy-ach": "^1.19.0",
```

Comment:
no ^

Reply:
oh, sorry. Maybe we could add a pre-commit hook for this...

Reply:
We can, but honestly I don't think we need to change this in PRs. Renovate does it automatically for us once merged.
Another trick is to use `yarn add cozy-ach -E` when adding the dependency.